src/views/welcome/WelcomeModal.scss

```diff
@@ -0,0 +1,27 @@
+.WelcomeModal {
+  &__grid {
+    margin-top: var(--pf-global--spacer--2xl);
+    margin-bottom: var(--pf-global--spacer--md);
+    margin-left: var(--pf-global--spacer--xl);
+    margin-right: var(--pf-global--spacer--xl);
```

Comment:
you can use the shortend of margin like so:
`margin: var(--pf-global--spacer--2xl) var(--pf-global--spacer--xl) var(--pf-global--spacer--md) var(--pf-global--spacer--xl);  /* top, right, bottom, left */`

another little improvement to that, you can also omit the value for left margin and will be as right margin:
`margin: var(--pf-global--spacer--2xl) var(--pf-global--spacer--xl) var(--pf-global--spacer--md); /* top, right+left, bottom`
from MDN docs: https://developer.mozilla.org/en-US/docs/Web/CSS/margin

Reply:
Tried, of course, but margin: `var(--pf-global--spacer--2xl) var(--pf-global--spacer--xl) var(--pf-global--spacer--md) var(--pf-global--spacer--xl);` simply didn't work and I haven't found the root cause of that problem. Spent a lot of time on that. Additionally, I think it is more readable like it is now.
Since the 1st option didn't work, I haven't tried what you suggest as the 2nd option, and directly went for the long version.

Reply:
Using the 2nd option - providing left+right value for margin as just one value did work. Thanks!